image/v3/modifying/drawing.md

```diff
@@ -126,9 +126,9 @@ $image = $manager->read('test.png');
 
 // draw a red ellipse with a blue border
 $image->drawEllipse(10, 10, function ($ellipse) {
-    $ellipse->size(180, 125); // width & height of ellipse
-    $ellipse->background('f00'); // background color
-    $ellipse->border('00f', 1); // border color & size
+    $ellipse->setSize(180, 125); // width & height of ellipse
+    $ellipse->setBackgroundColor('f00'); // background color
+    $ellipse->setBorder('00f', 1); // border color & size
```

Comment:
Please revert. These were correct calls of class `Intervention\Image\Geometry\Factories\EllipseFactory::class`.